README.md

```diff
@@ -26,16 +26,9 @@
 ├── .github/
 ├── public/
 ├── src
-│   ├── actions/
 │   ├── assets/
-│   ├── components/
+│   ├── features/
 │   ├── helpers/
```

Comment:
```suggestion
│   ├── utils
│   │   ├── helpers/
│   │   ├── constants.js
│   │   ├── (schemas.js ?)
```

Reply:
i am ok with the suggestion..  if we get some more 👍 i'll commit the suggestion

Reply:
I like `utils` better than `helpers`, but I don't think `constants.js` should be inside `utils` 😬

Reply:
I would keep it as it is, until we reach the point where we need to have global constants that are outside the specific features